app/controllers/posts_controller.rb

```diff
@@ -7,4 +8,21 @@ def index
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(params[:id])
+    @post.title = params[:title]
+    @post.text = params[:text]
+    @post.user_id = current_user.id
```

Comment:
- [ ] Just like in the comments controller, Add post_params as a private method which you will then pass to the create action. [Use this lesson as a recap](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).

Reply:
Thanks, I just fix it